server/datastore/cached_mysql/cached_mysql.go

```diff
@@ -2,51 +2,109 @@ package cached_mysql
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"reflect"
 	"time"
 
 	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/jinzhu/copier"
 	"github.com/patrickmn/go-cache"
 )
 
+const (
+	appConfigKey                      = "AppConfig:%s"
+	defaultAppConfigExpiration        = 1 * time.Second
+	packsHostKey                      = "Packs:host:%d"
+	defaultPacksExpiration            = 1 * time.Minute
+	scheduledQueriesKey               = "ScheduledQueries:pack:%d"
+	defaultScheduledQueriesExpiration = 1 * time.Minute
+	teamAgentOptionsKey               = "TeamAgentOptions:team:%d"
+	defaultTeamAgentOptionsExpiration = 1 * time.Minute
+)
+
+// Cloner represents any type that can clone itself. Used by types to provide a more efficient clone method.
+type Cloner interface {
```

Comment:
Looks like this is only used internally to detect implementation of this optional interface, should be unexported.

Reply:
Unless we expect external types to check proper implementation with the `var foo cahed_mysql.Cloner = (*Type)(nil)` thingy, but that's a pretty straightforward 1-method interface.

Reply:
Fixed in 5a2f37c4